runtime/src/main/java/dev/ebullient/micrometer/runtime/CompositeRegistryCreator.java

```diff
@@ -34,38 +24,13 @@
 public class CompositeRegistryCreator {
     private static final Logger log = Logger.getLogger(CompositeRegistryCreator.class);
 
+    public static CompositeMeterRegistry rootRegistry;
```

Comment:
Can we create a static setter (or something) for this so people know to look for who calls the setter to set this value? Either that, or have the producer reference/return a static value from the Recorder ..